client/app/bundles/course/admin/pages/CodaveriSettings/components/AssessmentList.tsx

```diff
@@ -0,0 +1,64 @@
+import { FC } from 'react';
+import { List, Typography } from '@mui/material';
+import { AssessmentProgrammingQuestionsData } from 'types/course/admin/codaveri';
+
+import Section from 'lib/components/core/layouts/Section';
+import Subsection from 'lib/components/core/layouts/Subsection';
```

Comment:
`Typography` and `Subsection` imports can be removed. I'm not sure why `jslint` doesn't catch this.